iotwifi/wpacfg.go

```diff
@@ -57,7 +57,7 @@ func NewWpaCfg(log bunyan.Logger, cfgLocation string) *WpaCfg {
 }
 
 // StartAP starts AP mode.
-func (wpa *WpaCfg) StartAP() {
+func (wpa *WpaCfg) StartAP() (*exec.Cmd) {
```

Comment:
Why the change in function signature? I don't see this response used in this PR anywhere.

Reply:
Nice catch - this is leftover from an earlier commit that I reverted.